source/5_security.rst

```diff
@@ -98,22 +98,53 @@ attack – recovery to an old known bad image.
 Measurment and Attestation
 ~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-Trusted Boot (measured boot)
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+OEM Component Measurement
+^^^^^^^^^^^^^^^^^^^^^^^^^
 
-The platform shall enable trusted boot. The early boot component shall
+The platform shall follow `TCG PC Client Platform Firmware Profile <https://trustedcomputinggroup.org/resource/pc-client-specific-platform-firmware-profile-specification/>'_
```

Comment:
What is the consequence? Is a TPM chip required now?

Reply:
> Please [remove the dot/period from the git commit message summary](https://cbea.ms/git-commit/).

Done.